### ISSUE TITLE
Raw() in WHERE resulting in quoted output

### ISSUE THREAD

**Issue by TCB13** [usmanhalalit/pixie#71]
Hello,

I was doing this: `$query->where("t.id", "IS", self::$mysql->raw("null"));`

If I try to debug the query, I get this:
`````
AND `t`.`id` IS null GROUP B...
`````
However when I run the query, it gives me a PDO Exception with the following:
`````
SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''null' GROUP BY `S...
`````

By inspecting MySQL's `general_log` I found out, Pixie or PDO are actually trying to escape the "null" value with single quotes:
`````
 AND `t`.`id` IS 'null' GROUP ...
`````

Why is this happening? Raw should not escape anything...

Thank you.

**Comment by neoascetic** [comment]
That's why there is `whereNull` query's method.

**Comment by TCB13** [comment]
@neoascetic well, my bad. Still there isn't any documentation about: `whereNull()`, `whereNotNull()`, `orWhereNull()`, `orWhereNotNull()`.

**Comment by neoascetic** [comment]
Yeah, docs need to be updated to reflect the latest changes. `raw()` doesn't work here because its value passed as parameter for binding and escaping is done by PDO, not by Pixie.

**Comment by TCB13** [comment]
Great, gonna do a PR on this doc update very soon.

**Comment by TCB13** [comment]
Fixed by #89
